src/SFA.DAS.PR.Application/EmployerRelationships/Queries/GetProviderEmployerRelationship/GetProviderEmployerRelationshipQueryResult.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SFA.DAS.PR.Domain.Entities;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.PR.Application.EmployerRelationships.Queries.GetProviderEmployerRelationship;
+
+[ExcludeFromCodeCoverage]
```

Comment:
This should not be excluded from coverage, we should have coverage for the operator.